projects/kit/components/tree/components/tree/tree.template.html

```diff
@@ -9,7 +9,7 @@
         </ng-container>
     </ng-container>
     <tui-tree
-        *ngFor="let child of children"
+        *ngFor="let child of children; trackBy: trackBy"
```

Comment:
You need to also pass it on to the nested `tui-tree` component

Reply:
ah yeah